generators/kafka/templates/src/main/java/package/web/rest/KafkaConfiguration.java.ejs

```diff
@@ -0,0 +1,63 @@
+<%#
+ Copyright 2013-2022 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+package <%= packageName %>.config;
+
+<%_ if (messageBrokerKafka) { _%>
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Supplier;
+import org.springframework.messaging.MessageHeaders;
+import org.springframework.messaging.support.GenericMessage;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+<%_ } _%>
+@Configuration
+public class KafkaConfiguration {
+
+    <%_ if (messageBrokerKafka) { _%>
+        @Bean
+        public Supplier<GenericMessage> binding(){
+            Map<String, Object> map = new HashMap<>();
+            map.put(MessageHeaders.CONTENT_TYPE, MimeTypeUtils.TEXT_PLAIN_VALUE);
+            MessageHeaders headers = new MessageHeaders(map);
+            return () ->{
+                return new GenericMessage<String>("kafka_supplier", headers);
+            };
+        }
+
+        @Bean
+        public Consumer<Message<String>> consume() {
+            return message -> {
+                log.debug("Got message from kafka stream: {}", message.getPayload());
+                emitters
+                    .entrySet()
+                    .stream()
+                    .map(Map.Entry::getValue)
+                    .forEach((SseEmitter emitter) -> {
+                        try {
+                            emitter.send(event().data(message.getPayload(), MediaType.TEXT_PLAIN));
+                        } catch (IOException e) {
+                            log.debug("error sending sse message, {}", message.getPayload());
+                        }   
+                });
+            };
+        }
+    
+    <%_ } _%>
+}
```

Comment:
Please ident the file.
Generate a project with this file and copy it here prettified.
```suggestion
}

```